### PR TITLE
Translate quickstart

### PR DESCRIPTION
**Pull request by 4demos**
# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you include a Data source and Documentation reference?
- [x] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [x] Did you check your descriptive content for spelling and grammar errors?
- [x] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [x] Does the PR contain a screenshot for each of your dashboards?
- [x] Do your screenshots show data?
- [x] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

File: quickstarts/aws/aws-translate/config.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+documentation:
+  - name: Amazon Translate installation docs
+    description: |
+      Monitor Amazon Translate by connecting AWS to New Relic.
+    url: https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-translate/
+keywords:
+  - aws
+  - amazon web services
+  - translate
+installPlans:
```
Comment (brammerl) [inline quickstarts/aws/aws-translate/config.yml:27]: Please remove this as we have deprecated `installPlans`

File: quickstarts/aws/aws-translate/config.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  - aws-translate
+dataSourceIds:
```
Comment (brammerl) [inline quickstarts/aws/aws-translate/config.yml:29]: Please create this data source as it does not currently exist or update this to refer to the proper data source. 

https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#data-sources

File: quickstarts/aws/aws-translate/config.yml (hunk 1 above)
Comment (4demos) [reply quickstarts/aws/aws-translate/config.yml]: removed

File: quickstarts/aws/aws-translate/config.yml (hunk 2 above)
Comment (4demos) [reply quickstarts/aws/aws-translate/config.yml:27]: added

**Review by brammerl — APPROVED** [review]

**Pull request merged**